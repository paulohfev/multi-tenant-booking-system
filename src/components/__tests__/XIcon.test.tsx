import { render } from '@testing-library/react'
import XIcon from '../XIcon'

describe('XIcon', () => {
  it('should match snapshot', () => {
    const { container } = render(<XIcon classNameStyling='w-6 h-6' />)
    expect(container).toMatchSnapshot()
  })
})
