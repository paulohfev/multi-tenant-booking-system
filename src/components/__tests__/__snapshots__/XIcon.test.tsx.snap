// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`XIcon should match snapshot 1`] = `
<div>
  <svg
    class="w-6 h-6"
    fill="none"
    stroke="currentColor"
    viewBox="0 0 24 24"
  >
    <path
      d="M6 18L18 6M6 6l12 12"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;
